### PR TITLE
Change summary, description, doc and name

### PR DESCRIPTION
**Pull request by pmaier1**
- Screenshot still missing.. @hurradieweltgehtunter

### CODE REVIEW

File: appinfo/info.xml [hunk 1]
```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0"?>
 <info>
 	<id>brute_force_protection</id>
-	<name>Brute Force Protection</name>
-	<description>ownCloud brute force protection app. It hardens ownCloud security by implementing brute force protection.</description>
-	<summary>ownCloud brute force protection app</summary>
+	<name>Brute-Force Protection</name>
+	<description>
+A brute-force attack consists of an attacker trying many user passwords with the hope of eventually guessing correctly and gaining access to a user account thereby. Frequently such attacks are conducted in an automated way which allows attackers to try many different combinations in a short time frame. In addition attackers may use dictionaries to reduce the amount of potential combinations to actual words ("dictionary attack"). Depending on the strength of a password you can estimate the amount of attempts and the respective time it will take to break it and gain access to user data. Administrators should be aware of this potential threat and take respective counter-measures.
```
Comment (phil-davis) [inline appinfo/info.xml]: ``thereby`` could be deleted. I just sounds like a superfluous word.

``take respective counter-measures``
maybe:
``take appropriate counter-measures``

**Comment by pmaier1** [comment]
Just realized: I changed the "name" adding a dash. That's an issue because "name" and the app folder need to match for the app to work, right?!

File: appinfo/info.xml [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
+
+The **Brute-Force Protection** extension allows administrators to specify a limit on the amount of unsuccessful login attempts. Once the limit is approached ownCloud will save the IP address ("ban") the unsuccessful login attempts were conducted from and restrict login for the combination of the user account in context and the IP address for a configurable time frame. With this an attacker is forced to either change their IP address or wait for the ban time to run-off in order to continue the attack. This best-practice approach increases the amount of time required to try a certain amount of combinations drastically making attacks unfeasible and mitigating risks to a minimum.
```
Comment (phil-davis) [inline appinfo/info.xml]: ``("ban")`` maybe goes better directly after ```save``

**Comment by pmaier1** [comment]
Thanks @phil-davis. Feedback incorporated.

**Comment by DeepDiver1975** [comment]
@pmaier1 you can Set any Name. Id is critical

**Comment by pmaier1** [comment]
Great, thx. So waiting for @hurradieweltgehtunter regarding screenshot and @karakayasemi for feedback on wording.

**Pull request merged**